### ISSUE TITLE
Printing out validation accuracy to the console (after each batch)?

### ISSUE THREAD

**Issue by moondra2017** [keras-team/keras#8862]
Sorry to post here, but I've already posted multiple places and so far no responses. 

I can't seem to find anything in the docs on how to print out the validation
accuracy after each batch, in addition to the training accuracy. 

I've currently posted on stackoverflow and reddit but so far no responses:

https://stackoverflow.com/questions/47930176/printing-out-the-validation-accuracy-to-the-console-for-every-batch-or-epoch-ke?noredirect=1#comment82831600_47930176

Thank you.

**Comment by mpariente** [comment]
The idea is that you go through you validation set after each epoch, not after each batch. 
If every batch, you had to evaluate the performances of the model on the whole validation set, you would loose a lot of time. 
I'll post the same answer on SO.

**Comment by ranjita-naik** [comment]
I achieved something similar by setting the epochs = 1 and then looping over the number_of_epochs as shown below

for epoch in range(no_of_epochs):
    print "Running epoch: %d" % epoch
    vgg.fit(batches, val_batches, epochs =1)
